### PR TITLE
Role and Permission CRUD

### PR DESCRIPTION
**Pull request by ezozafayzullayeva**

### CODE REVIEW

File: src/main/java/com/example/onefit/user/permission/dto/PermissionResponseDto.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PermissionResponseDto {
+    private UUID id;
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/user/permission/dto/PermissionResponseDto.java:13]: NotBlank qolib ketgan

File: src/main/java/com/example/onefit/user/permission/dto/PermissionUpdateDto.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.example.onefit.user.permission.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PermissionUpdateDto {
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/user/permission/dto/PermissionUpdateDto.java:10]: bu yerda ham NotBlank qolib ketgan

File: src/main/java/com/example/onefit/user/role/dto/RoleResponseDto.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class RoleResponseDto {
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/user/role/dto/RoleResponseDto.java:12]: NotBlank

File: src/main/java/com/example/onefit/user/role/dto/RoleUpdateDto.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.example.onefit.user.role.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class RoleUpdateDto {
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/user/role/dto/RoleUpdateDto.java:10]: NotBlank

**Review by mavjudbekdev — CHANGES_REQUESTED** [review]

**Review by mavjudbekdev — APPROVED** [review]

**Comment by ezozafayzullayeva** [comment]
Role Permission CRUD to'g'irlandi

**Comment by ezozafayzullayeva** [comment]
Role va Permission CRUD to'g'irlandi

**Pull request merged**